pipeline_live/data/sources/alpaca.py

```diff
@@ -26,18 +27,31 @@ def _get_stockprices(symbols, limit=365, timespan='day'):
 
     api = REST()
     timeframe = TimeFrame.Minute if timespan == "minute" else TimeFrame.Day
+    start = None
+
+    #If we need days we need to go back limit days, v1 used to handle this by itself
+    if timespan != 'minute':
+        start = api.get_clock().timestamp.date() - timedelta(days=limit)
```

Comment:
Not a request to change but weird how this is handled for minutes but not days?

Reply:
yeah I don't actually see a way to get minutes into this call unless you go to it manually so thats why this got dropped here; however you're right this could probably be just the same for minutes good catch :see_no_evil: